src/DoubleEndedQueueInterface.php

```diff
@@ -0,0 +1,156 @@
+<?php
+/**
```

Comment:
#33 added type hints throughout, so please update this interface by declaring `strict_types`.